Vertexvis/vertex-web-sdk
# [PLAT-2246] Additional box selection polish

## Summary

Adds a few more improvements to the `<vertex-viewer-box-query-tool>`, specifically:
- Adds a `dragCancelled` event to the model to indicate when a click that does not result in a volume intersection query occurs
- Adds a `clearAndSelect` operation type which allows for a default behavior of replacing existing selection

## Test Plan

- Verify that the default operation is `clearAndSelect` and it behaves as expected
- Verify that changing the operation type to `select` or `deselect` still behaves as expected
- Verify that a `dragCancelled` is emitted when clicking rather than dragging

## Release Notes

N/A

## Possible Regressions

Operation types

## Dependencies

N/A
